testsuite/features/step_definitions/common_steps.rb

```diff
@@ -436,7 +434,7 @@
   $server.run_until_ok("ls /var/cache/rhn/repodata/#{channel}/*updateinfo.xml.gz")
 end
 
-And(/^I push package "([^"]*)" into "([^"]*)" channel$/) do |arg1, arg2|
+When(/^I push package "([^"]*)" into "([^"]*)" channel$/) do |arg1, arg2|
```

Comment:
Aren`t these words (When, Then, And, ...) interchangeable when using the step in a feature fiel?

Reply:
Sure they are, but we must avoid using "And" in the step definition, as this keyword is too generic. Sometimes, by mistake, when developing using the IDE, I first write the sentence in .feature file then I auto generate the step definition from the IDE, but the IDE just pick as keyword the keyword wrote in the .feature file.
So, those are leftovers that must be fixed.

Reply:
Ok, so we should avoid `and` in the step definition.

Reply:
yes.